yb-voyager/src/srcdb/yugabytedb.go

```diff
@@ -351,7 +351,7 @@ func (yb *YugabyteDB) getExportedColumnsListForTable(exportDir, tableName string
 }
 
 // GetAllSequences returns all the sequence names in the database for the given schema list
-func (yb *YugabyteDB) GetAllSequences() []string {
+func (yb *YugabyteDB) GetAllSequences(_ []sqlname.NameTuple) []string {
```

Comment:
Shouldn't we also change the implementation for this to use FETCH_SEQUENCES_FOR_TABLE_LIST?

Reply:
yeah we can, though this function is never used as such but for correctness will do

Reply:
Before this change, this would be used, right? 
After this change, this is not used because in export-data-from-target, we now rely on the sequence list from the source? 

Is my understanding correct?

Reply:
Hmm, this is slightly concerning that we're not consistent in the offline and live part. 
For offline, we're using `GetAllSequences()` but for the debezium live part, we're using `GetColumnToSequenceMap`. 

Can we be consistent so that both the phases are using the same set of sequences?